p2p/libp2p/discovery/kadDhtDiscoverer.go

```diff
@@ -98,12 +101,37 @@ func (kdd *KadDhtDiscoverer) connectToInitialAndBootstrap() {
 		<-chanStartBootstrap
 
 		kdd.mutKadDht.Lock()
-		err := kdd.kadDHT.BootstrapWithConfig(ctx, cfg)
+		go func() {
+			i := 0
+			for {
+				if kdd.initc {
+					err := kdd.kadDHT.BootstrapOnce(ctx, cfg)
+					if err == kbucket.ErrLookupFailure {
+						// KDD: no more peers, Reconnect to initial list
+						chanRecInit := kdd.connectToOnePeerFromInitialPeersList(
+							kdd.refreshInterval,
+							kdd.initialPeersList)
+						<-chanRecInit
+
+					}
+				} else {
```

Comment:
Should improve code coverage on the else branch and in the functions: Pause, Resume